packages/headless/src/features/sort-criteria/criteria.test.ts

```diff
@@ -82,4 +84,66 @@ describe('sort-criterion functions', () => {
 
     expect(expression).toBe(expectedExpression);
   });
+
+  describe('#parseCriterionExpression', () => {
```

Comment:
Following on the comment in the other PR, I would prefer to have these tests in a file dedicated to the parser.